subprojects/docs/src/docs/userguide/custom_tasks.adoc

```diff
@@ -177,48 +184,49 @@ include::sample[dir="userguide/tasks/incrementalTask/kotlin",files="build.gradle
 
 NOTE: The code for this example can be found at **`samples/userguide/tasks/incrementalTask`** in the ‘-all’ distribution of Gradle.
 
-If for some reason the task is executed non-incrementally, e.g. by running with `--rerun-tasks`, all files are reported as `ADDED`, no matter of the previous state.
-Gradle automatically removes the previous outputs, so the incremental task only needs to process the given files.
+If for some reason the task is executed non-incrementally, for example by running with `--rerun-tasks`, all files are reported as `ADDED`, irrespective of the previous state.
+In this case, Gradle automatically removes the previous outputs, so the incremental task only needs to process the given files.
 
-For a simple transformer task like this, the task action simply needs to generate output files for any out-of-date inputs, and delete output files for any removed inputs.
+For a simple transformer task like the above exmaple, the task action simply needs to generate output files for any out-of-date inputs and delete output files for any removed inputs.
 
+[IMPORTANT]
+====
 A task may only contain a single incremental task action.
+====
 
 [[sec:which_inputs_are_considered_out_of_date]]
 === Which inputs are considered out of date?
 
 When there is a previous execution of the task, and the only changes since that execution are to incremental input file properties, then Gradle is able to determine which input files need to be processed (incremental execution).
-In this case, the link:{groovyDslPath}/org.gradle.work.InputChanges.html#org.gradle.work.InputChanges:getFileChanges(org.gradle.api.file.FileCollection)[InputChanges.getFileChanges] method returns details for all input files for the given property that were _added_, _modified_ or _removed_.
+In this case, the link:{groovyDslPath}/org.gradle.work.InputChanges.html#org.gradle.work.InputChanges:getFileChanges(org.gradle.api.file.FileCollection)[InputChanges.getFileChanges()] method returns details for all input files for the given property that were _added_, _modified_ or _removed_.
```

Comment:
When referencing to `getFileChanges`, is it obvious that the method takes an argument? Should we call that out as `InputChanges.getFileChanges(property)`?

Reply:
It depends whether the argument or arguments are relevant to whatever the text is describing. I think there's an argument for including the argument here as you suggest, since the text refers to "the given property".